config.yml

```
@@ -21,3 +21,6 @@ data-club-demo:
 partybal:
   gcs_bucket: partybal
   single_page_app: false
+numbers-that-matter-prototype:
```

Given that the URL is already protosaur.dev, I'd argue we don't need the `-prototype`

```suggestion
numbers-that-matter:
```